tests/integ/test_basic.py

```diff
@@ -59,20 +167,20 @@ class Meta:
 
 
 def test_stream_read(engine):
-    class MyModel(BaseModel):
+    class MyStreamReadModel(BaseModel):
```

Comment:
Is this because we don't clean up tables between tests, so the same name tries to re-use an existing table?  We could add a `table_name_template` function or hook into the `before_create_table` signal to nonce table names per-test.